src/mach_dep.c

```diff
@@ -339,6 +336,10 @@ setup()
 	 */
 	COFF();
 	ocb = set_ctrlb(0);
+#ifndef ROGUE_NO_X11
+	if (getenv("DISPLAY"))
```

Comment:
maybe `char *env = NULL; if (env = getenv("DISPLAY") && *env) ...` just to make sure `DISPLAY` is both set **and** not-empty? I believe this would prevent the one-time error in PuTTY. It would at least allow `DISPLAY='' src/rogue`